docs/UserGuide.md

```diff
@@ -200,39 +195,39 @@ This command displays a list of all guests in GuestBook.
 
 This command edits an existing guest in GuestBook.
 
-The guest specified at the `INDEX` is the guest you wish to edit. The existing values will be updated to the values
-that you have inputted. The values that you did not specify to edit will remain the same.
+The guest specified at the `INDEX` is the guest you wish to edit. The existing values will be **updated** to the new values
+you provide. The values that you did not specify to edit will **remain unchanged**.
```

Comment:
Should we bold the "did not specify" as well?

Reply:
Changed!